packages/@okta/vuepress-site/.vuepress/nav/reference.js

```diff
@@ -37,6 +37,7 @@ module.exports = [
     { title: 'Registration Hook', path: '/docs/reference/registration-hook/'},
     { title: 'Token Hook', path: '/docs/reference/token-hook/'},
     { title: 'Okta Expression Language', path: '/docs/reference/okta-expression-language/'},
+    { title: 'Password Hook', path: '/docs/reference/password-hook/'},
```

Comment:
Is there a reason why Password Hook comes after the OEL, instead of with the other "X Hook" reference items?

Reply:
I assume it should be in alphabetical order, so, have now hanged those other hook links to be in order.